packages/next/src/views/ForgotPassword/index.tsx

```diff
@@ -70,6 +81,7 @@ export const ForgotPassword: React.FC<AdminViewProps> = ({ initPageResult }) =>
       <Form
         action={`${serverURL}${api}/${userSlug}/forgot-password`}
         // handleResponse={handleResponse}
+        initialState={initialState}
```

Comment:
I dont think we need to use about autoLogin at all in this file. You could instead just create the initialState inline here, with default unfilled text.

Reply:
Yep - forgot to clear that out.